src/main/java/com/artipie/api/UsersRest.java

```diff
@@ -51,6 +52,24 @@ public void init(final RouterBuilder rbr) {
         rbr.operation("putUser")
             .handler(this::putUser)
             .failureHandler(this.errorHandler(HttpStatus.INTERNAL_SERVER_ERROR_500));
+        rbr.operation("deleteUser")
+            .handler(this::deleteUser)
+            .failureHandler(this.errorHandler(HttpStatus.INTERNAL_SERVER_ERROR_500));
+    }
+
+    /**
+     * Removes user.
+     * @param context Request context
+     */
+    private void deleteUser(final RoutingContext context) {
+        try {
+            this.users.remove(context.pathParam(RepositoryName.UNAME));
+            this.cache.invalidateAll();
```

Comment:
Can we to narrow the 'try'-block to catch exactly exception rom 'users.remove(...)'?
I mean:
        try {
            this.users.remove(context.pathParam(RepositoryName.UNAME));
        } catch (final IllegalStateException err) {
            Logger.error(this, err.getMessage());
            context.response().setStatusCode(HttpStatus.NOT_FOUND_404).end();
        }
        this.cache.invalidateAll();
        context.response().setStatusCode(HttpStatus.OK_200).end();

Reply:
@olenagerasimova thanks!